.github/workflows/tox.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         pyversion_no_dot="${{ matrix.python-version }}"
         pyversion_no_dot="${pyversion_no_dot/./}"
         pip install tox
-        tox -v -r -e py${pyversion_no_dot} .
+        tox run -v -e py${pyversion_no_dot} -- .
```

Comment:
To make sure you always get the expected behavior using a modern `tox` version, I would recommend to bump `tox` requirement to `tox>=4.0`, both in `requirements-dev.txt` and the `min_version` setting in `tox.ini` (see: https://tox.wiki/en/4.11.4/config.html#min_version)